config/grafana/manual-provisioning/contact-points.json

```diff
@@ -0,0 +1,35 @@
+{
+    "template_files": {
+        "rocketchat - alert": "{{ define \"rocketchatAlert\" }}\n# {{ .Labels.alertname }}\n## Labels\n{{ range .Labels.SortedPairs }}\n- {{ .Name }}: {{ .Value }}\n{{ end }}\n## Annotations\n{{ if gt (len .Annotations) 0 }}\n  {{ range .Annotations.SortedPairs }}\n- {{ .Name }}: {{ .Value }}\n  {{ end }}\n  {{ end }}\n## Links\n  {{ if gt (len .SilenceURL ) 0 }}\n- [Silence alert]({{ .SilenceURL }}) \n  {{ end }}\n  {{ if gt (len .DashboardURL ) 0 }}\n- [Go to dashboard]({{ .DashboardURL }})\n  {{ end }}\n{{ end }}",
+        "rocketchat - messages": "{{ define \"rocketchatMessages\" }}\n  {{ if gt (len .Alerts.Firing) 0 }}\n    {{ range .Alerts.Firing }} {{ template \"rocketchatAlert\" .}} {{ end }}\n  {{ end }}\n  {{ if gt (len .Alerts.Resolved) 0 }}\n    {{ range .Alerts.Resolved }} {{ template \"rocketchatAlert\" .}} {{ end }}\n  {{ end }}\n{{ end }}"
+    },
+    "alertmanager_config": {
+        "route": {
+            "receiver": "jankaritech rocketchat"
+        },
+        "templates": [
+            "rocketchat - messages",
+            "rocketchat - alert"
+        ],
+        "receivers": [
+            {
+                "name": "jankaritech rocketchat",
+                "grafana_managed_receiver_configs": [
+                    {
+                        "name": "jankaritech rocketchat",
+                        "type": "slack",
+                        "disableResolveMessage": false,
+                        "settings": {
+                            "recipient": "#test",
+                            "text": "{{ template \"rocketchatMessages\" . }} ",
+                            "token": "foo"
```

Comment:
this field does not exist in the config I see in the WebUI of grafana I think the authentication purely uses the URL